vendor/src/github.com/docker/libcontainer/namespaces/nsenter/nsenter_unsupported.go

```diff
@@ -1,3 +1,2 @@
-// +build !linux !cgo
```

Comment:
We cannot be changing `vendor/` here -- if this change needs to be made, it should be made in https://github.com/docker/libcontainer directly and then the vendored copy updated via `project/vendor.sh`.